analytics/pubstack/helpers.go

```diff
@@ -0,0 +1,59 @@
+package pubstack
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/prebid-server/analytics"
+	"net/http"
+	"net/url"
+)
+
+type RequestType string
+
+const (
+	HEALTH = "/v1/health"
```

Comment:
Capitalization of constants is not advised in Go due to the capitalization of the first character denoting package scope. This should be `health` or `healthPath`. 

I realize older code in this repo is also using this anti-pattern, so I understand where you got it from.